src/java/org/apache/cassandra/db/compaction/UnifiedCompactionStrategy.java

```diff
@@ -529,6 +529,25 @@ List<Level> getLevels()
         return getLevels(getSSTables(), UnifiedCompactionStrategy::isSuitableForCompaction);
     }
 
+    /**
+     * @return a list of the levels in the compaction hierarchy, that also includes SSTables that
+     * are currently undergoing compaction. This is used only for table stats so we can have a consistent
+     * snapshot of the levels.
+     */
+    @VisibleForTesting
```

Comment:
This method is being used in other places along with tests, may be we can remove `@VisibleForTesting` annotation